src/Auth0Client.ts

```diff
@@ -1232,6 +1192,32 @@ export class Auth0Client {
     };
   }
 
+  private async _saveEntryInCache(entry: CacheEntry) {
+    const { id_token, decodedToken, ...entryWithoutIdToken } = entry;
+
+    await this.cacheManager.setIdToken(
+      this.options.clientId,
+      entry.id_token,
+      entry.decodedToken
+    );
+    await this.cacheManager.set(entryWithoutIdToken);
+  }
+
+  private async _getIdTokenFromCache() {
+    const audience = this.options.authorizationParams?.audience || 'default';
+    const scope = getUniqueScopes(this.scope);
```

Comment:
I don't _think_ it'll make much difference but could we reference `this.scope` directly rather than running it through `getUniqueScopes`?

Reply:
Ohh, that's a left over! thanks